pkg/util/singularityconf/config.go

```diff
@@ -323,4 +325,11 @@ mksquashfs procs = {{ .MksquashfsProcs }}
 # Allow to share same images associated with loop devices to minimize loop
 # usage and optimize kernel cache (useful for MPI)
 shared loop devices = {{ if eq .SharedLoopDevices true }}yes{{ else }}no{{ end }}
+
+# IMAGE DRIVER: [STRING]
+# DEFAULT: Undefined
+# Image driver allows to specify a driver installed by a plugin responsible
```

Comment:
How about...

This option specifies the name of an image driver provided by a plugin that will be used to handle image mounts. If the 'enable overlay' option is set to 'driver'  the driver name specified here will also be used to handle overlay mounts. If the driver name specified has not been registered via a plugin installation the run-time will abort.